### DISCUSSION TITLE
Research Boards UI/docs feedback

### DISCUSSION THREAD

**Discussion by danshearer** [Leantime/leantime#569] [General]
I don't believe there is a relationship between the columns in the Research Boards, but at first I thought it would make sense for there to be a connection. I suggest making it clear in the docs that this is kanban-style, with no intention of reading from left to right across the columns.

There's a lot to learn about the Leantime mental model, so reducing the trip hazards like this helps a bit :-)

Thanks for the app

**Comment by marcelfolaron** [comment]
Thank you for the feedback. I agree, the research boards are somewhat difficult to grasp at first. I'll work on the docs some more
